spec/grape/validations/params_scope_spec.rb

```diff
@@ -466,6 +466,20 @@ def initialize(value)
       end.to raise_error(Grape::Exceptions::UnknownParameter)
     end
 
+    it 'DOES NOT raise an error if the dependent parameter is a Hash' do
+      expect do
+        subject.params do
+          optional :d, type: Integer
```

Comment:
I am nitpicking, but is this `optional :d` needed for the spec?

Reply:
I did to populate declared_params with something more than a Hash.
Just removed as I believe there are other specs covering thath.